crates/db_views/src/combined/modlog_combined_view.rs

```diff
@@ -355,6 +380,16 @@ impl ModlogCombinedQuery {
       }
     }
 
+    let is_subscribed = community_actions::followed.is_not_null();
+    query = match self.listing_type.unwrap_or(ListingType::All) {
+      ListingType::All => query,
+      ListingType::Subscribed => query.filter(is_subscribed),
+      ListingType::Local => query
+        .filter(community::local.eq(true))
+        .filter(community::hidden.eq(false).or(is_subscribed)),
+      ListingType::ModeratorView => query.filter(community_actions::became_moderator.is_not_null()),
```

Comment:
The new filter. The only caveat here is that when doing a non-`All` filter, its going to filter by community, meaning some modlog types are filtered out.

Reply:
Make a comment about that somewhere (so that it goes into openapi docs).

Reply:
K, I'll add comments to all the `GetModlog` api fields now.